src/audio/rtnr/rtnr.c

```diff
@@ -450,16 +420,22 @@ static int32_t rtnr_set_value(struct comp_dev *dev, struct sof_ipc_ctrl_data *cd
 {
 	uint32_t val = 0;
 	int32_t j;
+	struct comp_data *cd = comp_get_drvdata(dev);
 
 	for (j = 0; j < cdata->num_elems; j++) {
 		val |= cdata->chanv[j].value;
-		comp_info(dev, "rtnr_set_value(), value = %u", val);
+		comp_dbg(dev, "rtnr_set_value(), value = %u", val);
 	}
 
-	if (val)
-		rtnr_set_process(dev);
-	else
-		rtnr_set_bypass(dev);
+	if (val) {
```

Comment:
why not just use `!val` for everything here so we can remove the if?

Reply:
@cujomalainey Fixed this. Thank you!